Kajabi/sage-lib
# [Sage-493] Sage 3 - Button Updates

Closes #493 

_FYI: this work is going in as a separate version bump (note the merging branch is `main`)_

## Description

Updates styling to Sage 3.0 specs:

* Adds a variation that styles the `secondary` button like the designed tertiary button ([for later consideration](https://github.com/Kajabi/sage-lib/issues/844))
* Aligns focus rings, `color` and `background-color` to designs
* Subtle buttons: Remove small variation in documentation example in favor of disabled
* Add clearer headings for each section of the button types
* Overall documentation improvements

## Screenshots
### Buttons

|  Before  |  After  |
|--------|--------|
| ![Screen Shot 2021-09-28 at 9 06 11 AM](https://user-images.githubusercontent.com/24237393/135105524-d2234b23-ba31-4d94-8369-fd34ad703476.png) | ![Screen Shot 2021-09-30 at 10 58 38 AM](https://user-images.githubusercontent.com/24237393/135490714-71c7ff6c-09dd-472a-9801-2f52d9d111b1.png)  |
---

### Additional options
|  Before  |  After  |
|--------|--------|
| ![Screen Shot 2021-09-28 at 11 06 37 AM](https://user-images.githubusercontent.com/24237393/135124557-bf4613c6-d81d-4575-8310-5f010c1b0c2c.png) | ![Screen Shot 2021-09-28 at 11 06 47 AM](https://user-images.githubusercontent.com/24237393/135124577-9a2fb1ae-ee05-4830-9a62-38989d4affe8.png) |


## Testing in `sage-lib`
1. Navigate to http://localhost:4000/pages/component/button
2. Observe new updates and verify in alignment with specs


## Testing in `kajabi-products`
1. (**BREAKING**) Button updates to the usage of `secondary` can potentially alter the UI throughout the app
   - [ ] Buttons are everywhere
